CODING_STANDARDS.md

```diff
@@ -13,6 +13,19 @@ New code should follow these rules, and old code should be changed over time to
 
 We adhere to the `pep8` standard, using `flake8`, `pylint` and `mypy` to catch further programming errors or standard violations. Type hints must be used everywhere.
 
+## Solidity
+
+The Solidity code follows the following conventions:
+
+- Contracts are named following `PascalCase`. Contract names and contract files have the same name (e.g. the contract `MyContract` is implemented in `MyContract.sol`). Same applies for libraries.
+- Testing contract names end with `_test`.
+- Documentation code comments follow [NatSpec](https://docs.soliditylang.org/en/v0.7.1/natspec-format.html#natspec).
+- Variable names and argument names are in `snake_case` (e.g. `my_variable`) except *state variables* which are prefixed with `_` (e.g. `_my_state_var`).
+- Constant names are in `UPPER_SNAKE_CASE`.
+- Function names are in `snake_case`.
```

Comment:
`Method names`, maybe?

Reply:
I don't have strong preferences (methods tend to denote "object's functions" in OOP, but a method truly is a function). However, they seem to stick to "function" in solidity (see e.g. here: https://docs.soliditylang.org/en/v0.7.1/contracts.html?highlight=method#functions) rather than "methods" for contracts.

Reply:
For some reason, I thought functions *had* to be on contracts, in which case "methods" would seem more suitable. But I now see from the specs that functions can actually be standalone, so "functions" is more natural.